pymks/fmks/func.py

```diff
@@ -276,7 +276,11 @@ def make_da(func):
     """
 
     def wrapper(arr, *args, **kwargs):
-        return func(rechunk(arr.shape, arr), *args, **kwargs)
+        if isinstance(arr, np.ndarray):
+            chunks = arr.shape
```

Comment:
Overall, everything looks good to me too. I am still thinking about if there is a way to automatically chunk the NumPy array.  Because right now, if the input is NumPy array, we are making it a single chunk and leveraging distributed computing. I guess it is not an issue, they need to know how to use it properly

Reply:
That's an interesting idea. We can do [this](https://docs.dask.org/en/latest/array-chunks.html#automatic-chunking). I was kind of thinking that if a user is using a Numpy array then they are assuming that it is all a serial calculation, but I'm not sure. It's this explicit versus implicit behavior. I think fmks is supposed to be more explicit so I think it's probably better to keep it fully serial probably, but I'm not 100% on that.

Reply:
Actually, we should probably be using `auto` chunking in the notebooks regardless of what we choose for the above.